src/ChildProcessUtilities.js

```diff
@@ -33,7 +36,8 @@ export default class ChildProcessUtilities {
 
   static execSync(command, opts) {
     const mergedOpts = objectAssign({
-      encoding: "utf8"
+      encoding: "utf8",
```

Comment:
is this the default value? this was already present. perhaps for an older version of node?